modules/odc_con_customizing-a-developer-catalog-or-its-sub-catalogs.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assembly:
+//
+// * web_console/customizing-the-web-console.adoc
+
+:_content-type: CONCEPT
+
+[id="odc_con_customizing-a-developer-catalog-or-its-sub-catalogs_{context}"]
+= Developer catalog and sub-catalog customization
+
+As a cluster administrator, you have the ability to organize and manage the Developer catalog or its sub-catalogs by enabling or disabling the sub-catalog types or disabling the entire developer catalog. In some situations, customers do not want their users to have access to the Developer catalog or one or more of its sub-catalogs.
+
+`developerCatalog: types` includes the following parameters that you must define in a snippet to use them in the YAML view:
+
+* `state`: Defines if a list of developer catalog types should be enabled or disabled.
+* `enabled`: Defines a list of developer catalog types (sub-catalogs) that are visible to users.
+* `disabled`: Defines a list of developer catalog types (sub-catalogs) that are not visible to users.
+
+You can enable or disable the following developer catalog types (sub-catalogs) can be enabled or disabled using the YAML view or the form view.
```

Comment:
Required: This needs to be re-worded - maybe remove the "can be enabled or disabled".

Reply:
Fixed.